### PR TITLE
Codecov fix via curl

### PR DESCRIPTION
**Pull request by jdtoombs**

### CODE REVIEW

File: .github/workflows/api-dotnetcore.yml [hunk 1]
```diff
@@ -71,17 +71,20 @@ jobs:
...
-          token: ${{env.codeCov-token}}
-          # Path to coverage file to upload
-          file: ${{env.working-directory}}/tests/unit/TestResults/coverage.opencover.xml
-          # Flag upload to group coverage metrics (e.g. unittests | integration | ui,chrome)
-          flags: unittests
-          # Environment variables to tag the upload with (e.g. PYTHON | OS,PYTHON)
-          env_vars: C#
-          # Specify whether or not CI build should fail if Codecov runs into an error during upload
-          fail_ci_if_error: true
+        run: |
```
Comment (devinleighsmith) [inline .github/workflows/api-dotnetcore.yml:74]: is there an issue number or something detailing what is broken?

Comment (devinleighsmith) [reply .github/workflows/api-dotnetcore.yml:74]: What I mean is, is this a known issue with the codecov-action?

Comment (jdtoombs) [reply .github/workflows/api-dotnetcore.yml:74]: Sort of, I have a few threads of people with the same issue with no resolution. Will post below. Think my most recent changes screwed something up though. Added some flags and messed up the syntax.

Comment (jdtoombs) [reply .github/workflows/api-dotnetcore.yml:74]: https://github.com/codecov/codecov-action/issues/58

Comment (devinleighsmith) [reply .github/workflows/api-dotnetcore.yml:74]: you can probably test the curl from your local machine once you generate the testing artifacts.

**Comment by llorllale** [comment]
Heads up: updating to codecov-action@1.0.13 seemed to solve the issue: https://github.com/hyperledger/aries-framework-go/pull/2206

**Review by Fosol — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**